mindone/models/latte.py

```diff
@@ -0,0 +1,312 @@
+import logging
+import os
+
+import mindspore as ms
+from mindspore import Tensor, nn, ops
+from mindspore.common.initializer import XavierUniform, initializer
+
+from .dit import (
+    DiTBlock,
+    FinalLayer,
+    LabelEmbedder,
+    PatchEmbed,
+    TimestepEmbedder,
+    constant_,
+    get_2d_sincos_pos_embed,
+    normal_,
+    xavier_uniform_,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class Latte(nn.Cell):
+    def __init__(
+        self,
+        input_size=32,
```

Comment:
Since this part is being integrated into the library, it would be nice for the code to be well typed and documented.

Reply:
Thanks for your advice. It is improved.